app/config/StringsConfig.scala

```diff
@@ -8,4 +8,5 @@ class StringsConfig {
   val supportLandingDescription = config.getString("supportLanding.description")
   val contributionsLandingDescription = config.getString("contributionsLanding.description")
   val subscriptionsLandingDescription = config.getString("subscriptionsLanding.description")
+  val digitalPackLandingDescription = config.getString("digitalPackLanding.description")
```

Comment:
`config.getString` returns the config item or will throw an exception (see: https://lightbend.github.io/config/latest/api/com/typesafe/config/Config.html#getString-java.lang.String-).

Ideally we want to wrap in Option here to catch that exception and return None or Some if the config item is present. This prevents you having to wrap with Some below and assuming the config item is there. 

what you do with the Option here if it is None is a design choice. You can default to some other String, fail loudly with sys.error if it's core to the application running (not in this instance) or propagate the Option throughout as you have done.

Reply:
Good shout. We shall read those strings as Option[String]. The None case is already handled in the template main.scala.html.